src/test/java/FFMPredictorTest.java

```diff
@@ -34,4 +38,19 @@ void hash(String[] input, int[] expected) {
     void sigmoid(double input, double expected) {
         assertEquals(FFMPredictor.sigmoid(input), expected);
     }
+
+    static List<Arguments> predictTestCases() {
+        List<FeatureValue> l = ((Stream<FeatureValue>) IntStream.rangeClosed(1, 16).mapToObj(i -> new FeatureValue(String.valueOf(i), i))).collect(Collectors.toList());
+        FeatureValue[] arr = l.toArray(FeatureValue[]::new);
```

Comment:
Variable "l" (lower L) is too similar to "1" (one). Choose better name. Believe me I thought for a moment that maybe integer literal ("1") has toArray method...